### DISCUSSION TITLE
Stack Bar Chart shows truncated/overlapped dataLabels

### DISCUSSION THREAD

**Discussion by ShreyaMishra03** [apexcharts/apexcharts.js#4138] [Q&A]
Hi Team,
Is there a way to show DataLabels only if the length of the horizontal bar is greater than a certain length (say 5 or 10). Currently it is getting overlapped, due to short bar length. 

Tried adding formatter but it completely hides the bar, instead of just the label inside.
Here is how it is looking currently

![image](https://github.com/apexcharts/apexcharts.js/assets/108660618/ef561d27-06a7-4823-a927-4c0d25d73d04)

**Comment by junedchhipa** [comment]
Please provide a reproducible example on codepen

**Reply by ShreyaMishra03** [reply]
https://codesandbox.io/p/sandbox/apx-bar-stacked-forked-mzwry3?file=%2Fsrc%2Fapp%2Fapp.component.ts%3A49%2C30

Here we can see the overlapping and truncation issue.
![Uploading image.png…]()
